Library/Formula/fb-infer.rb

```diff
@@ -0,0 +1,53 @@
+class FbInfer < Formula
```

Comment:
:+1: for `infer`; this is how upstream calls it. They probably used fbinfer.com because infer.com is already taken.